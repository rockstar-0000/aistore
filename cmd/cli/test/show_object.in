echo "0123456789" > $OBJECT_1.txt
ais bucket create ais://$BUCKET_1 // IGNORE
ais object put $OBJECT_1.txt ais://$BUCKET_1/object_1.txt // IGNORE

ais show object ais://$BUCKET_1/object_2.txt // FAIL "object_2.txt" not found in ais://$BUCKET_1"
ais show object ais://$BUCKET_2/object_1.txt // FAIL "Bucket "ais://$BUCKET_2" does not exist"

ais show object ais://$BUCKET_1/object_1.txt
ais show object ais://$BUCKET_1/object_1.txt --props=size,version
ais show object ais://$BUCKET_1/object_1.txt --props=all

ais show object aiss://$BUCKET_1/object_1.txt // FAIL "invalid backend provider "aiss""

ais object put $OBJECT_1.txt ais://$BUCKET_1 // IGNORE
ais object set-custom ais://$BUCKET_1/object_1.txt key1=val1 // IGNORE
ais object set-custom ais://$BUCKET_1/object_1.txt key222=val222 // IGNORE
ais show object ais://$BUCKET_1/object_1.txt --props=all
