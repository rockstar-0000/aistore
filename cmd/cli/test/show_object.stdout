PROPERTY         VALUE
^atime(.*)$
checksum         xxhash[05967d5390ac53b0]
name             ais://$BUCKET_1/object_1.txt
size             11B
PROPERTY         VALUE
size             11B
version          1
PROPERTY         VALUE
^atime(.*)$
checksum         xxhash[05967d5390ac53b0]
^copies\s*1(.*)$
custom           -
ec               -
^location(t.+:mp.+)$
name             ais://$BUCKET_1/object_1.txt
size             11B
version          1
PROPERTY         VALUE
^atime(.*)$
checksum         xxhash[05967d5390ac53b0]
^copies\s*1(.*)$
custom           map[key1:val1 key222:val222]
ec               -
^location(t.+:mp.+)$
name             ais://$BUCKET_1/object_1.txt
size             11B
version          1
